frontend/public/style/_overrides.scss

```diff
@@ -203,3 +203,12 @@ tags-input .autocomplete .suggestion-item em {
     border-bottom: 1px solid $table-border-color;
   }
 }
+
+.blank-slate-pf {
```

Comment:
If we are adding this override, we should be able to remove at least some of the styles here:

https://github.com/spadgett/console/blob/0b573b72939e2500af46e22e2a11ef921c8a871c/frontend/public/components/catalog/_catalog.scss#L172-L180

Reply:
I actually don't see that class being used anywhere in catalog, so those can probably be removed altogether unless I'm missing something.